src/web.config

```diff
@@ -5,8 +5,12 @@
       <handlers>
         <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModuleV2" resourceType="Unspecified" />
       </handlers>
-      <aspNetCore requestTimeout="00:02:30" processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" stdoutLogEnabled="false">
-        <environmentVariables />
+      <aspNetCore requestTimeout="00:02:30" processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" stdoutLogEnabled="false" hostingModel="InProcess">
+        <environmentVariables>
+          <environmentVariable name="ASPNETCORE_HTTPS_PORT" value="44360" />
+          <environmentVariable name="ASPNETCORE_ENVIRONMENT" value="local" />
+          <environmentVariable name="COMPLUS_ForceENC" value="1" />
+        </environmentVariables>
```

Comment:
Are these changes needed? From what I remember these are added in by Visual Studio when running the application locally but shouldn't be used in live?